### ISSUE TITLE
Invalid XML

### ISSUE THREAD

**Issue by cornips** [SlavaYurthev/MultipleLayeredNavigation-M2#2]
When compiling Magento 2.2.5:
```
  Invalid XML in file /vendor/sy/multiple-layered-navigation/etc/events.xml:
  Element 'event', attribute 'name': [facet 'pattern'] The value 'sy.copyright' is not accepted by the pattern '[a-zA-Z0-9_]+'.
  Line: 10

  Element 'event', attribute 'name': 'sy.copyright' is not a valid value of the atomic type 'eventName'.
  Line: 10

  Element 'event', attribute 'name': Warning: No precomputed value available, the value was either invalid or something strange happend.
  Line: 10
```

Just like https://github.com/SlavaYurthev/Slider-M2/issues/9, with same fix

**Comment by mjoraid** [comment]
@cornips  Indeed. I'm getting the same issue after I upgraded this extension to `V 0.3.0.`
What I did is change the file:
`sy/multiple-layered-navigation/etc/events.xml` 
Edit line:
`<event name="sy.copyright">`
to
`<event name="sy_copyright">`
Another option is to delete:
`sy/multiple-layered-navigation/etc/events.xml` 
all together. 
and this seems to stop triggering the error. I think the developer is trying to add a copyright notice somewhere and the name `sy.copyright` is not a valid syntax.
